lib/http.js

```diff
@@ -79,23 +99,26 @@ function httpRequest(sdk, options) {
     });
 }
 
-function get(sdk, url, saveResponse) {
+function get(sdk, url, options) {
   url = util.isAbsoluteUrl(url) ? url : sdk.options.url + url;
-  return httpRequest(sdk, {
+  var getOptions = {
     url: url,
-    method: 'GET', 
-    dontSaveResponse: !saveResponse
-  });
+    method: 'GET'
+  };
+  util.extend(getOptions, options);
+  return httpRequest(sdk, getOptions);
 }
 
-function post(sdk, url, args, dontSaveResponse) {
+function post(sdk, url, args, options) {
   url = util.isAbsoluteUrl(url) ? url : sdk.options.url + url;
-  return httpRequest(sdk, {
+  var postOptions = {
     url: url,
     method: 'POST',
     args: args,
-    dontSaveResponse: dontSaveResponse
-  });
+    cacheState: true
```

Comment:
Hmm, why is cacheState true for post? When would we actually cache a post response?